frappe/core/report/prepared_report_analytics/prepared_report_analytics.json

```diff
@@ -0,0 +1,30 @@
+{
+ "add_total_row": 0,
+ "columns": [],
+ "creation": "2024-12-18 11:58:00.693755",
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "filters": [],
+ "idx": 0,
+ "is_standard": "Yes",
+ "letterhead": null,
+ "modified": "2024-12-18 11:58:00.693755",
+ "modified_by": "Administrator",
+ "module": "Core",
+ "name": "Prepared Report Analytics",
+ "owner": "Administrator",
+ "prepared_report": 0,
+ "ref_doctype": "Prepared Report",
+ "report_name": "Prepared Report Analytics",
+ "report_type": "Script Report",
+ "roles": [
+  {
+   "role": "System Manager"
+  },
+  {
+   "role": "Prepared Report User"
```

Comment:
Remove this, these reports will only make sense to developers.

Reply:
Removed